index.html

```diff
@@ -367,6 +368,29 @@ <h2>
             </ol>
           </dd>
         </dl>
+        <dl data-link-for="MediaDevices" data-dfn-for="MediaDevices">
+          <dt>
+            <dfn>preferredLabel</dfn>
+          </dt>
+          <dd>
+            <p>This settable attribute allows an application to inform the user
+            agent of a meaningful {{MediaStreamTrack/label}} to apply to the
+            tracks that would result if the tab in which the application is 
+            running is captured.<p>
+            <p>Only the top-level document may set or read this value. Attempts
```

Comment:
Do you know if there are other ways for a doc to know if it's a top-level doc?
Anyway, I think saying that only the preferredLabel of the top-level document can be used as label would be enough. WDYT?

Reply:
I think `window === window.top` is an easier way for a document to check whether it's the top-level document, so we'd not be exposing that bit of information for the first time.

Your suggestion makes sense to me; I will enact the change.